sig-security/security-audit-2021/RFP.md

```diff
@@ -127,3 +127,12 @@ The audit should result in the following deliverables, which will be made public
 ### Do we need to use our own hardware and infrastructure or should we use a cloud?
 
 Strong preference would be for the vendor to provide their own infrastructure or use a public cloud provider, just NOT a managed offering like GKE or EKS. The reasoning is to prevent accidentally auditing a cloud provider's kubernetes service instead of kubernetes/kubernetes. Depending on the scope and approach, it may make sense to use a local cluster (e.g. kind) for API fuzzing and anything that doesn't impact the underlying OS, and is an easy to use repeatable setup (see Methodology above).
+
+### What is the intellectual property ownership of the report and all work product?
+
+The report must be licensed under the Creative Commons Attribution 4.0 International Public License (CC BY 4.0) based on [section 11.(f) of the Cloud Native Computing Foundation (CNCF) Charter](https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy).
+Separately, any code used needs to be under the Apache License, version 2.0. Please refer to [sections 11.(e) and (d) in the CNCF Charter](https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy).
```

Comment:
I'm concerned the word "used" here might be interpreted to prevent a firm from using any proprietary tooling in the process of the audit.

Can we say "any code released with or as part of the report" or something similar?

Reply:
@tabbysable I'll admit freely that these snippets were direct from LF folks; that said wrt. the sections (d) and (e) referenced, really only "outbound code" part seems to apply. I'm not precisely clear what "outbound code" means - it's not actually mentioned in the apache text, but the closest google hit was: https://opensource.guide/legal/ "inbound (from contributors) and outbound (to other contributors and users)".  So since a tool "used" doesn't seem to fit the outbound (or inbound) definition, I would tend to agree.  I'm fine with your text.